src/cadenza.test.ts

```diff
@@ -152,6 +158,29 @@ describe('Given a Cadenza JS client instance', () => {
     });
   });
 
+  describe('Given a Cadenza JS client instance with different origin', () => {
+    let iframe: HTMLIFrameElement;
+    let cad: CadenzaClient;
+
+    beforeEach(() => {
+      iframe = document.createElement('iframe');
+      cad = cadenza(BASE_URL, { iframe, targetOrigin: EXTERNAL_LINK_KEY });
+    });
+
+    it('Includes given origin param', () => {
+      cad.show(EMBEDDING_TARGET_ID);
+      const expectedKey = `webApplicationLink=${encodeURIComponent(
+        EXTERNAL_LINK_KEY.externalLinkId,
+      )}`;
+      const expectedKeyRepository = `webApplicationLinkRepository=${encodeURIComponent(
```

Comment:
The encoding should not be necessary for the test IDs, right? Proposal: Just interpolate the variables in the URL-string below, so that it's more clear how the expected URL looks like.